repos/standard.proj

```diff
@@ -27,12 +28,13 @@
 
     <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildArguments)</BuildCommand>
 
-    <BuildPackagesCommand></BuildPackagesCommand>
     <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
     <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
-    <PackagesOutput>$(ProjectDirectory)/artifacts/packages/$(Configuration)/Shipping</PackagesOutput>
+    <ShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</ShippingPackagesOutput>
+    <NonShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/NonShipping/</NonShippingPackagesOutput>
```

Comment:
There's no new .NET Standard for 5.0, should we remove this project?

Reply:
There will no longer be new .NET Standard builds going forward?

Reply:
See https://github.com/dotnet/runtime/issues/2294#issuecomment-579963951 for some context:

> We're working through this. My expectation is that [dotnet/standard] will become similar to CoreFX/CoreCLR in that it's only used for servicing older releases and moving forward it'll be replaced by the runtime repo. However, this doesn't mean we need to build netstandard assets from the runtime repo. My expectation is that the `net5` TFM will subsume both netcoreapp and netstandard.

Reply:
Removed.